webapp/src/components/Parcel/Parcel.js

```diff
@@ -19,6 +21,8 @@ export default class Parcel extends React.PureComponent {
     isConnecting: false,
     isLoading: false,
     ownerOnly: false,
+    notOwnerOnly: false,
+    withPublications: false,
```

Comment:
I use it to redirect the user if it is the owner, eg you shuold not request a mortgage for your own parcels.

`notOnlyOwner` = all users except owner.

Maybe I should find a better name

`ownerNotAllowed`, `notOwner`?

Reply:
I use it to redirect the user if it is the owner, eg you shuold not request a mortgage for your own parcels.

`notOnlyOwner` = all users except owner.

Maybe I should find a better name

`ownerNotAllowed`, `notOwner`?

Reply:
@NicoSantangelo @nachomazzara maybe not for this PR, but I would rather have an ```onAccess: (parcel, wallet) => boolean``` callback that receives parcel and wallet and return a boolean (either the user can access or not) and then let each component that uses `<Parcel>` be the owner of that business logic, otherwise we keep adding rules to `<Parcel>` that are too use-case-specific.